Loadable/ParticlesDisplay/Logic/vtkSlicerParticlesDisplayLogic.cxx

```diff
@@ -124,7 +124,7 @@ vtkMRMLParticlesNode* vtkSlicerParticlesDisplayLogic::AddParticlesNode (const ch
   vtkNew<vtkMRMLModelStorageNode> storageNode;
 
   storageNode->SetFileName(filename);
-  if (!storageNode->ReadData(particlesNode.GetPointer()) != 0)
+  if (storageNode->ReadData(particlesNode.GetPointer()) != 0)
```

Comment:
@rjosest have a look at the logic here and see if you agree...

Reply:
Good catch. That was a bug. Thanks for contributing.